### PR TITLE
Bundle fixes and documentation improvements

### PR DESCRIPTION
**Pull request by llllllllll**
- make `quantopian-quandl` the default bundle
- update documentation to reflect new default bundle
- update documentation to make `quandl` bundle as deprecated
- improve error message when no benchmark is provided

### CODE REVIEW

File: docs/source/bundles.rst [hunk 1]
```diff
@@ -119,30 +117,22 @@ Default Data Bundles
...
-   $ QUANDL_API_KEY=<api-key> zipline ingest -b quandl
+   $ zipline ingest -b quantopian-quandl
+   $ zipline ingest
 
-though we may still run ``ingest`` as an anonymous quandl user (with no API
-key). We may also set the ``QUANDL_DOWNLOAD_ATTEMPTS`` environment variable to
-an integer which is the number of attempts that should be made to download data
-from quandl's servers. By default ``QUANDL_DOWNLOAD_ATTEMPTS`` will be 5, meaning
-that we will retry each attempt 5 times.
+These commands should only take a few seconds to download the data.
```
Comment (richafrank) [inline docs/source/bundles.rst:130]:
```suggestion
Either command should only take a few seconds to download the data.
```

File: zipline/algorithm.py [hunk 2]
```diff
@@ -140,6 +140,13 @@
 AttachedPipeline = namedtuple('AttachedPipeline', 'pipe chunks eager')
 
 
+class NoBenchmark(ValueError):
+    def __init__(self):
+        super().__init__(
```
Comment (richafrank) [inline zipline/algorithm.py:145]: Reminder for py2 compat (for this release)

File: zipline/utils/run_algo.py [hunk 3]
```diff
@@ -189,32 +189,46 @@ def choose_loader(column):
...
+                'handle_data': handle_data,
+                'before_trading_start': before_trading_start,
+                'analyze': analyze,
+            } if algotext is None else {
+                'algo_filename': getattr(algofile, 'name', '<algorithm>'),
+                'script': algotext,
+            }
+        ).run()
+    except NoBenchmark:
+        raise _RunAlgoError(
```
Comment (richafrank) [inline zipline/utils/run_algo.py:220]: Nice use of `_RunAlgoError`. I hadn't remembered this existed.

**Comment by richafrank** [comment]
xref https://github.com/quantopian/zipline/issues/2603

File: docs/source/bundles.rst [hunk 4]
```diff
@@ -119,30 +117,22 @@ Default Data Bundles
 Quandl WIKI Bundle
 ``````````````````
 
-By default zipline comes with the ``quandl`` data bundle which uses quandl's
-`WIKI dataset <https://www.quandl.com/data/WIKI>`_. The quandl data bundle
-includes daily pricing data, splits, cash dividends, and asset metadata. To
-ingest the ``quandl`` data bundle we recommend creating an account on quandl.com
```
[the remark this answers is not in the document]
Comment (llllllllll) [reply docs/source/bundles.rst:125]: I don't think this bundle will still work, so we can remove it from the docs.

**Pull request merged**